tests/integration/targets/letsencrypt/test.yml

```diff
@@ -0,0 +1,34 @@
+- name: create the certificate for example.com
+  hosts: localhost
+  collections:
+    - t_systems_mms.letsencrypt
+  roles:
+    - letsencrypt
+  vars:
+    domain:
+      certificate_name: "example.com"
+      zone: "example.com"
+      email_address: "ssl-admin@example.com"
+      subject_alt_name:
+        - example.com
```

Comment:
As we lately had issues with wildcards at the AutoDNS API, should we test this right away with a second playbook? (https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/pull/23/files#diff-42714dee14ae9428f690fb01f7c9f3f4c51c1d31ab7cf52ca68ed4735553bff3R16)

Same thinking as in the linked comment applies here: maybe not needed

Reply:
Yeah, not needed. We'd just test that our pebble test was correct, nothing else.